app/controllers/administrative-units/administrative-unit/governing-bodies/index.js

```diff
@@ -0,0 +1,18 @@
+import Controller from '@ember/controller';
+
+export default class AdministrativeUnitsAdministrativeUnitGoverningBodiesIndexController extends Controller {
+  get governingBodies() {
```

Comment:
It seems this isn't used anymore. The template uses the model directly. So I would just remove the controller if it's not used for anything else.